src/lib/expression/lqp_select_expression.cpp

```diff
@@ -57,6 +58,16 @@ bool LQPSelectExpression::is_nullable() const {
   return lqp->column_expressions()[0]->is_nullable();
 }
 
+bool LQPSelectExpression::is_correlated() const {
```

Comment:
`return !arguments.empty()` is sufficient.

Reply:
Hmm... that is confusing. Where is it guaranteed / documented that arguments only holds external parameters, not prepared-statement ones?

Reply:
I agree, #1111

Reply:
changed. approve?